package.json

```diff
@@ -1013,6 +1013,10 @@
                 {
                     "command": "aws.renderStateMachineGraph",
                     "when": "false"
+                },
+                {
+                    "command": "aws.dev.openMenu",
+                    "when": "aws.isDevMode"
```

Comment:
I'd prefer that there's just a single registered command. Otherwise it's just noise.